package.json

```diff
@@ -166,6 +166,14 @@
                     "default": 30,
                     "minimum": 1,
                     "description": "Controls the maximum number of saved revisions allowed for a given file"
+                },
+                "local-history.excludeFiles": {
+                    "type": "object",
+                    "markdownDescription": "Configure glob patterns for excluding files and folders from creating Local History. The glob includes `**./local-history` as default. Read more about glob patterns [here](https://code.visualstudio.com/docs/editor/codebasics#_advanced-search-options).",
+                    "default": {
+                        "**/.local-history": true
+                    },
```

Comment:
I think the default should support `.local-history` and anything defined under `files.exclude` (ex: _node_modules_)